### PR TITLE
SP-1507: Updated docs in developer section

### PR DESCRIPTION
**Pull request by iamsamwhite**
@oboreo

### CODE REVIEW

File: section/doc/tactic-developer/developer/setting-up-the-schedule-event-trigger/index.txt [hunk 1]
```diff
@@ -12,29 +13,38 @@ NOTE: Ensure that the <process_count> is set to 3: <process_count>3</process_cou
...
-Data (Drop Down)
-schedule
-scripts/script_path
-To Run Trigger Daily (24 Hour Clock):
+[width="100%", cols="30%, 30%, 40%”,]
+|========================================
+|Event| Script Path | Data (Drop Down) 
+|schedule | scripts/script_path | To Run Trigger Daily (24 Hour Clock):
+
+---------------------------------------------
```
Comment (oboreo) [inline section/doc/tactic-developer/developer/setting-up-the-schedule-event-trigger/index.txt]: Does that actually draw a table?

File: section/doc/tactic-developer/developer/packaging-python-scripts-into-a-plugin/index.txt [hunk 2]
```diff
@@ -3,30 +3,32 @@ TACTIC - Packaging Python Scripts into a Plugin
...
+8.  With the plugins installed, all of the classes contained within the scripts that were loaded with the plugin are accessible from the Script Editor in TACTIC. All you would need to do is know where the classes are located within the files. For example:
 
+-------------------------------------------------------
 from example_scripts.python_scripts import python_class
-
+-------------------------------------------------------
 
 

 Updating Plugins with New and Modified Scripts
```
Comment (menechel) [inline section/doc/tactic-developer/developer/packaging-python-scripts-into-a-plugin/index.txt]: This needs to be denoted as a title like the "Creating Plugin for Scripts"

File: section/doc/tactic-developer/developer/packaging-python-scripts-into-a-plugin/index.txt [hunk 3]
```diff
@@ -3,30 +3,32 @@ TACTIC - Packaging Python Scripts into a Plugin
 
 *Creating Plugin for Scripts*
 
-1) Prepare a folder or folder structure containing all of the python scripts you would like to include in the plugin. 
+1.  Prepare a folder or folder structure containing all of the python scripts you would like to include in the plugin. 
 
-The scripts can be contained within any folder level in the hierarchy. The plugin will allow you to have access to every one of those scripts.
 
-2) Copy your folder of scripts.
```
Comment (menechel) [inline section/doc/tactic-developer/developer/packaging-python-scripts-into-a-plugin/index.txt]: When generating the books, this comes out as item #1 and the rest follow from here i.e. #2, #3, etc. This should be item #2. It looks like it's set correctly here, but please make sure it comes out correctly in the generated books.

File: section/doc/tactic-developer/developer/watch-folder-service/index.txt [hunk 4]
```diff
@@ -7,26 +7,33 @@ Currently the implementation will create an entry per file.  Subsequent drops of
...
+b)  Base Dir - The name of the directory that will be "watched". This is where you drop the files to be checked in to TACTIC. i.e. /home/tactic/drop. 
++
+c)  Search Type - The sType or table that the assets or files will be checked in to i.e. sample_project/media
++
+d)  Process (optional) - The pipeline process that the asset will be checked into. By default, the asset is checked into the "publish" process, if no process is specified. This is sufficient when no pipeline processes have been defined, or the asset doesn't need to be checked into a certain process. Otherwise, the process needs to be specified i.e. publish
+
+6.  Save the entry by selecting the floppy disk icon in the shelf.
+7.  If a new Watch Folder entry is added, restart the TACTIC service. To do this, if you SSH into the server and you are logged in as "root", navigate to the TACTIC install and type, "service tactic restart". This will restart the TACTIC service and the Watch Folder will be ready for use.
 
 Limitations
```
Comment (menechel) [inline section/doc/tactic-developer/developer/watch-folder-service/index.txt]: Maybe underline or italicize this based on common subsection conventions throughout the rest of the docs.

File: section/doc/tactic-developer/developer/watch-folder-service/index.txt [hunk 5]
```diff
@@ -45,23 +54,26 @@ Options:
...
+-----------------------------------------------
 
 By default, the Watch Folder will drop assets into the “assets” folder of the TACTIC installation location or /home/tactic/assets on a Virtual Machine, where they are seen by TACTIC. The assets will be stored under the “assets” folder according to the defined directory and file naming conventions. However, there is a way to ingest the assets into a different directory location other than the “assets” folder. The instructions below will describe two required actions needed to set up the watch folders to ingest to different directories:
-How to set up the Web Server Configuration and TACTIC Configuration so that the new directories can be seen by the Web Server and TACTIC and the assets can still be previewed in TACTIC
-How to set up the watch folder to ingest to different directory locations
+1.  How to set up the Web Server Configuration and TACTIC Configuration so that the new directories can be seen by the Web Server and TACTIC and the assets can still be previewed in TACTIC
+2.  How to set up the watch folder to ingest to different directory locations
 
 
 Configuring the Web Server and TACTIC
```
Comment (menechel) [inline section/doc/tactic-developer/developer/watch-folder-service/index.txt]: Bold or italicize as a subsection.

File: section/doc/tactic-developer/developer/watch-folder-service/index.txt [hunk 6]
```diff
@@ -94,37 +111,49 @@ An alias and the absolute path for the device directory you added will need to b
...
++
 	OR
++
 	service apache2 restart (Debian/Ubuntu Linux)
-	
-Then, to restart TACTIC:
++	
+        Then, to restart TACTIC:
++
 	service tactic restart
```
Comment (menechel) [inline section/doc/tactic-developer/developer/watch-folder-service/index.txt]: For this Section 1.6, please make sure the numbering of each step is coming out correctly in the documents. It seems as if they are numbered correctly here, but not showing properly when the books are generated.

File: section/doc/tactic-developer/developer/watch-folder-service/index.txt [hunk 7]
```diff
@@ -135,12 +164,13 @@ Directory Setup with Watch Folder
```
Comment (menechel) [inline section/doc/tactic-developer/developer/watch-folder-service/index.txt]: Bold or italicize "Directory Setup with Watch Folder" as subsection

File: section/doc/tactic-developer/developer/setting-up-the-schedule-event-trigger/index.txt (hunk 1 above)
Comment (iamsamwhite) [inline section/doc/tactic-developer/developer/setting-up-the-schedule-event-trigger/index.txt]: yes it draws a table

**Pull request merged**